### DISCUSSION TITLE
Can this support 1.20 armor trims when I upgrade my world?

### DISCUSSION THREAD

**Discussion by acropolis914** [Hipposgrumm/armor-trims#18] [General]
Like when I change versions will it automatically change to the vanilla trims or something?

**Comment by Hipposgrumm** [comment]
No, it doesn't. I don't believe this is possible.

**Comment by Hipposgrumm** [comment]
The recent update changed the format of the trim tag to be the same as vanilla for any _newly trimmed_ items. This also allows it to work with on servers with ViaBackwards.
